frontend/e2e/tests/inquiry.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { test } from "@playwright/test"
+import PageManager from "../page-objects/pageManager"
+
+test.describe("Inquiry Management Suite", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/")
+  })
+
+  test.afterEach(async ({ page }) => {
+    await page.close()
+  })
+
+  test("TC_001 Verify the presence of AddInquiry button on Inquiry Management page", async ({
```

Comment:
for Jest-style tests, this different naming convention is totally fine. The key thing to think about is that a test should only ever fail for one reason, and that reason needs to be obvious from test name.